src/docs/reference/modules/Liquid/README.md

```diff
@@ -559,6 +559,36 @@ Removes key from HttpContext.Items collection
 
 `{% httpcontext_remove_items "Item1" %}`
 
+### TrackingConsent
```

Comment:
```suggestion
### Tracking Consent Cookie
```

Reply:
The header is the name of the liquid property, like e.g. `HttpContext` etc.